src/__tests__/ceramic_cas.ts

```diff
@@ -0,0 +1,60 @@
+import {buildServicesFromConfig, Services} from "../index";
+import {AnchorStatus, CeramicApi} from "@ceramicnetwork/common"
+
+const registerChangeListener = function (doc: any): Promise<void> {
+    return new Promise(resolve => {
+        doc.on('change', () => {
+            resolve()
+        })
+    })
+}
+
+describe('Ceramic<->CAS integration', () => {
+    jest.setTimeout(1000 * 60 * 5) // 5 minutes
+    let ceramic: CeramicApi
+
+    beforeAll(async () => {
+        const services: Services = await buildServicesFromConfig();
+        ({ceramic} = services);
+    });
+
+    afterAll(async () => {
+        await ceramic.close()
+    });
+
+    test("basic crud", async () => {
+        // Test document creation
+        console.log("Creating document")
```

Comment:
My comment about using `describe` on your other pr applies here

We should do 
```js
describe('creating a tile document', () => {
  test('should persist its content', () => {
        const initialContent = { foo: 'bar' }
        const doc = await ceramic.createDocument('tile', {content: initialContent})
        expect(doc.content).toEqual(initialContent)
    })
})
```

This will also make it easier to do `test.only` or `test.skip` when we are debugging

Reply:
Given that there's only 1 test here, soon to be 2 with the other PR, I'm not really sure what value adding another nesting layer of "describe" gains us?  We'd wind up with a describe block with only a single test in it, I'm not sure that gains us anything?

Reply:
It seems like you are testing multiple things in a single block. You are testing that newly created documents have the correct anchor statuses, and that updated documents have the correct anchor statuses. Seems like you can break this down into 4 sections at least.

Maybe it is better as
```
describe(
  test(
  test(
  test(
  test(
```

It should be easier to read the testing output also if we have a problem with one piece of this its easier to isolate.

Reply:
Hmm, but the test of document updates would have to repeat all the logic from the test of document creation to set up the test state properly.  So whatever test is last would basically wind up looking exactly like the test I have now

Reply:
Ok I kinda see what you mean but I'm unclear on what we are actually testing here. I think the test is that the anchor service anchors correctly when we create and update documents?

Can we at least change `test("basic crud"` to be more precise? I think that will help. I would probably still split this up based on which part of crud is being tested. Here I guess it is just create and update.

Reply:
I renamed "basic crud" to "basic crud is anchored properly".
I still left it as one test though.  Making a separate test for update would require creating a document to update again, and waiting for it to be anchored, which slows the tests down, and duplicates all the logic of the "create" test.  I'm generally very on board with smaller, more focused tests, I just don't think it really is possible in this case.